addons/stock_account/models/stock.py

```diff
@@ -648,6 +666,8 @@ def _create_account_move_line(self, credit_account_id, debit_account_id, journal
                 ref = 'Revaluation of %s (negative inventory)' % ref
             elif self.env.context.get('forced_quantity') is not None:
                 ref = 'Correction of %s (modification of past move)' % ref
+        if not ref and self.inventory_id:
+            ref = self.inventory_id.name
```

Comment:
you don't have this field anymore

Reply:
Yes we have : https://github.com/odoo/odoo/blob/a765a38edbfd01bb886a65b2ea3ab93c85966497/addons/stock/models/stock_move.py#L131

Reply:
you're right, self is a move here, sorry